app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/triage/multipledisposals/asset_type_for_non_uk_residents.scala.html

```diff
@@ -72,56 +75,78 @@
-      )
-    )
-}
-
-@mainTemplate(title = title, userType = request.userType, hasErrors = hasErrors) {
-
-    @backLinkComponent(backLink)
+@selected = @{form.value.map(_.map(toIndex)).getOrElse(List.empty)}
 
-    @if(hasErrors) {
-        @errorSummary(form, Some(s"$key$userKey"))
-    }
+@checkBoxes = @{List(
+  CheckboxItem(
+    id = Some(messages(s"$key-0")),
+    content = Text(messages(s"$key.Residential")),
+    hint = Some(Hint(content = Text(messages(s"$key.Residential.helpText")))),
+    value = "0",
+    checked = selected.contains(0)
+  ),
+  CheckboxItem(
+    id = Some(messages(s"$key-1")),
+    content = Text(messages(s"$key.NonResidential")),
+    hint = Some(Hint(content = Text(messages(s"$key.NonResidential.helpText")))),
+    value = "1",
+    checked = selected.contains(1)
+  ),
+  CheckboxItem(
+    id = Some(messages(s"$key-2")),
+    content = Text(messages(s"$key.MixedUse")),
+    hint = Some(Hint(content = Text(messages(s"$key.MixedUse$userKey.helpText")))),
+    value = "2",
+    checked = selected.contains(2)
+  ),
+  CheckboxItem(
+    id = Some(messages(s"$key-3")),
+    content = Text(messages(s"$key$userKey.IndirectDisposal")),
+    hint = Some(Hint(content = Text(messages(s"$key.IndirectDisposal.helpText")))),
+    value = "3",
+    checked = selected.contains(3)
+  )
+)}
 
-    @caption(messages("triage.caption"))
 
-    @formWrapper(triage.routes.MultipleDisposalsTriageController.assetTypeForNonUkResidentsSubmit(), 'novalidate -> "novalidate") {
+@layout(
+  title,
+  hasErrors = hasErrors,
+  backLinkUrl = Some(backLink.url)
+) {
+  @if(hasErrors) {
+    @errorSummary(form, Some(s"$key$userKey"))
+  }
 
-        @checkboxGroup(
-            fieldId = key,
-            options = options,
-            selected = form.value.map(_.map(toIndex)).getOrElse(List.empty),
-            helpText = Some(Html(messages(s"$key.helpText"))),
-            label = pageHeading(title),
-            error = form.error(key),
-            customErrorKey = Some(s"$key$userKey")
-        )
+  <span class="govuk-caption-xl">
+    @messages("triage.caption")
+  </span>
 
-        @submitButton(messages(if(isAmend) "button.continue" else if (hasCreatedDraftReturn) "button.saveAndContinue" else "button.continue"))
-    }
+  @formWrapper(triage.routes.MultipleDisposalsTriageController.assetTypeForNonUkResidentsSubmit(), 'novalidate -> "novalidate") {
+    @viewHelpers.govukCheckboxes(Checkboxes(
+      fieldset = Some(Fieldset(
+        legend = Some(Legend(
+          content = Text(title),
+          classes = "govuk-fieldset__legend--l",
+          isPageHeading = true
+        ))
+      )),
+      hint = Some(Hint(
+        content = Text(messages(s"$key.helpText"))
+      )),
+      errorMessage = form.error(key).map(e => ErrorMessage(
+        content = Text(messages(s"$key$userKey.${e.message}")),
+        visuallyHiddenText = Some(messages("generic.error"))
+      )),
+      name = s"$key[]",
+      items = checkBoxes
+    ))
```

Comment:
could you add an idPrefix using key e.g. `idPrefix = Some(key)` to ensure we get checkbox ids using this key in a predictable way